code/extensions/che-remote/package.json

```diff
@@ -113,6 +113,11 @@
         {
           "command": "che-remote.command.openDocumentation",
           "group": "remote_40_che_navigation@30"
+        },
+        {
+          "command": "che-remote.command.stopWorkspaceAndRedirectToDashboard",
+          "group": "remote_40_che_navigation@11",
+          "when": "che-remote.openshift-console-enabled"
```

Comment:
I think the value for the `when` field is not correct

Reply:
Fixed